atlasdb-client/src/test/java/com/palantir/atlasdb/schema/SchemaMetadataTest.java

```diff
@@ -57,6 +62,16 @@ public void canSerializeAndDeserializeSchemaMetadataWithMultipleTables() {
                 .build());
     }
 
+    @Test
+    public void orderingOfCleanupRequirementEnumIsCorrect() {
```

Comment:
I'd also put a test with reverse sort. This is too easy to satisfy with compare that just returns 0.

Reply:
Good point. New version cleanup requirements aren't orderable though so we won't need this anymore :)